BookPlayer/AppDelegate.swift

```diff
@@ -59,6 +59,9 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         // register document's folder listener
         self.setupDocumentListener()
 
+        // register for CarPlay
+        self.setupCarPlay()
```

Comment:
this should be called inside: https://developer.apple.com/documentation/carplay/cpapplicationdelegate/2968287-application

Reply:
That method is only available in iOS 12. Should we consider raising the target?

Reply:
mmmmm a week ago I think I read it reached 50% adoption rate. There were other reasons we were considering raising the target (I don't remember which ones now though 🤔). Honestly I lean towards raising the target